ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -150,11 +150,12 @@
 
         #createCheckBoxItem(value, itemLabel) {
             // This doesn't cater for optionLink __links as toggelablelink component is hidden
-            //TOFIX: Rich text in aria-label
+            let richScreenReaderText = `${this._model.label.value}:  ${itemLabel}`;
```

Comment:
again should only be done if label is rich

Reply:
we by default consider options to be rich text as per crispr spec. https://git.corp.adobe.com/livecycle/af2-docs/blob/gh-pages/schema/adaptive-form-properties.schema.yaml#L145